liquidsoap.spec

```diff
@@ -117,12 +133,21 @@ exit 0
 %systemd_postun_with_restart liquidsoap@.service
 
 %files
-%{_exec_prefix}/bin/liquidsoap
-%{_unitdir}/liquidsoap@.service
-%config/etc/liquidsoap/radio.liq.example
-%config/etc/logrotate.d/liquidsoap
-%{_exec_prefix}/lib/liquidsoap/%{version}/
+%{_bindir}/%{name}
+%{_unitdir}/%{name}@.service
+%config%{_sysconfdir}/%{name}/radio.liq.example
+%config(noreplace)%{_sysconfdir}/logrotate.d/%{name}
+%{_exec_prefix}/lib/%{name}/%{version}/
```

Comment:
Can't this be replaced with `%{_libdir}/%{name}/%{version}/`, or are there problems with `lib64`?

Reply:
The upstream  installer currently places these into lib and not lib64. Right now this triggers `W: only-non-binary-in-usr-lib`. I've decided to keep them in `%{_exec_prefix}/lib/%{name}/` since that is where the docs tell end users to look.